tests/test_environment.py

```diff
@@ -0,0 +1,30 @@
+import glob
+import os.path as osp
+from textwrap import dedent
+import unittest
+
+from hpcbench.campaign import ReportNode
+from . import DriverTestCase
+
+
+class TestEnvironment(DriverTestCase, unittest.TestCase):
+    EXPECTED_SHELL_SCRIPT_PRELUDE = dedent("""\
+        #!/bin/sh
+
+        module purge
+        module load foo/bar
+        export foo=bar
+        """)
+
+    def test(self):
```

Comment:
We should create another fake benchmark which defines in its execution matrix two env variables. one which is overwritten by the user in the yaml, and one that is not, ideally the yaml will also define two env variables, one that overwrites a var in the benchmark and one that is added.. then we have it all covered.